e2e/utils/global-setup.ts

```diff
@@ -0,0 +1,53 @@
+import { join } from 'path';
+import { ChildProcess, execSync, spawn } from 'child_process';
+
+export default async function () {
+  let port = 4873;
+  global.nxLocalRegistryStorage = join(
+    process.cwd(),
+    'tmp/local-registry/storage',
+    process.env.NX_TASK_TARGET_PROJECT ?? ''
+  );
```

Comment:
followup: I don't think we actually need the project in the path here right? It can be any tmp directory.a